src/sparsezoo/v2/test.md

```diff
@@ -0,0 +1,111 @@
+# API User pathway
+
+## Download one file
+
+### Through the string arguments
+```python
+from sparsezoo import Model
+
+stub = "zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-moderate"
+stub = stub + "?" + "recipe=transfer_learn"
+# will read all the files from the server, except all the recipe files. The only recipe 
+# file that will be "seen" by the `model` is "transfer learn"
+model = Model(path=stub)
+model.download() # optional: may be called to download all the files, but is not required (see below). Will be omitted in further examples
+recipes: str = model.recipes.downloaded_path() # will automatically download the file (if not already downloaded) and return the path
+...
+```
+```python
+from sparsezoo import Model
+
+stub = "zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-moderate"
+stub = stub + "?" + "checkpoint=preqat"
+# will read all the files from the server, except for the full training folder. Training folder
+# will be identical to prequat checkpoint directory
+model = Model(path=stub)
+training: str = model.training.downloaded_path() 
+...
+```
+### Through the API purely
+```python
+from sparsezoo import Model
+
+stub = "zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-moderate"
+model = Model(path=stub)
+# ALTERNATIVE ONE: filtering on download
+model.recipes.download(recipe="transfer_learn")
+recipe: str = model.recipes.downloaded_path()
+# ALTERNATIVE TWO: filtering on .downloaded_path()
+recipe: str = model.recipes.downloaded_path(recipe="transfer_learn")
+```
+
+```python
+from sparsezoo import Model
+
+stub = "zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-moderate"
+model = Model(path=stub)
+# ALTERNATIVE ONE: filtering on download
+model.training.download(checkpoint="preqat")
+training: str = model.training.downloaded_path()
+# ALTERNATIVE TWO: filtering on .downloaded_path()
+training: str = model.training.downloaded_path(checkpoint="preqat")
+```
+
+## Download multiple chosen files (also in parallel)
+
+### Through the string arguments
+```python
+from sparsezoo import Model
+
+stub = "zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-moderate"
+# choose the files through the string arguments
+stub = stub + "?" + "recipe=transfer_learn" + "&" + "checkpoint=preqat" + "&" + "recipe=original"
```

Comment:
per URL standards, param names must be unique -- overwrite with the last if this is not the case